Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/Services/IAnalyticsService.cs

```diff
@@ -10,10 +13,22 @@ public interface IAnalyticsService
     {
         void Identify(string userId, JsonObject? traits = null);
 
+        /// <summary>
+        ///     To track an event you have to call identify first
+        /// </summary>
         void Track(string eventName, JsonObject? properties = null);
```

Comment:
Can we make an assertion inside Rust implementation of this plugin, that Identify was already called?